src/components/exchange/ConnectDevice.vue

```diff
@@ -63,7 +63,7 @@
         <v-col cols="10" class="d-flex justify-end ma-0 py-0 pl-0">
           <v-btn v-if="sendBitcoinState === 'idle' || sendBitcoinState === 'error'"
                  rounded color="#00B520" width="110"
-                 :disabled="sendBitcoinState === 'error'"
+                 :disabled="sendBitcoinState === 'error' || walletName ==='Liquality'"
```

Comment:
I think the button should be hidden rather than disabled if the wallet is not an HW wallet